### ISSUE TITLE
[SOLVED] ValueError: This ORT build has ['TensorrtExecutionProvider', 'CUDAExecutionProvider', 'CPUExecutionProvider']

### ISSUE THREAD

**Issue by fj44dn** [Gourieff/sd-webui-reactor#105]
### First, confirm

- [X] I have read the [instruction](https://github.com/Gourieff/sd-webui-reactor/blob/main/README.md) carefully
- [X] I have searched the existing issues
- [X] I have updated the extension to the latest version

### What happened?

```
Error running postprocess_batch: /home/studio-lab-user/microsoftexcel/extensions/sd-webui-reactor/scripts/reactor_faceswap.py
Traceback (most recent call last):
  File "/home/studio-lab-user/microsoftexcel/modules/scripts.py", line 496, in postprocess_image
    script.postprocess_image(p, pp, *script_args)
  File "/home/studio-lab-user/microsoftexcel/extensions/sd-webui-reactor/scripts/reactor_faceswap.py", line 362, in postprocess_image
    result, output, swapped = swap_face(
  File "/home/studio-lab-user/microsoftexcel/extensions/sd-webui-reactor/scripts/reactor_swapper.py", line 279, in swap_face
    source_face, wrong_gender, source_age, source_gender = get_face_single(source_img, face_index=source_faces_index[0], gender_source=gender_source)
  File "/home/studio-lab-user/microsoftexcel/extensions/sd-webui-reactor/scripts/reactor_swapper.py", line 198, in get_face_single
    face_analyser = copy.deepcopy(getAnalysisModel())
  File "/home/studio-lab-user/.conda/envs/default/lib/python3.9/copy.py", line 172, in deepcopy
    y = _reconstruct(x, memo, *rv)
  File "/home/studio-lab-user/.conda/envs/default/lib/python3.9/copy.py", line 270, in _reconstruct
    state = deepcopy(state, memo)
  File "/home/studio-lab-user/.conda/envs/default/lib/python3.9/copy.py", line 146, in deepcopy
    y = copier(x, memo)
  File "/home/studio-lab-user/.conda/envs/default/lib/python3.9/copy.py", line 230, in _deepcopy_dict
    y[deepcopy(key, memo)] = deepcopy(value, memo)
  File "/home/studio-lab-user/.conda/envs/default/lib/python3.9/copy.py", line 146, in deepcopy
    y = copier(x, memo)
  File "/home/studio-lab-user/.conda/envs/default/lib/python3.9/copy.py", line 230, in _deepcopy_dict
    y[deepcopy(key, memo)] = deepcopy(value, memo)
  File "/home/studio-lab-user/.conda/envs/default/lib/python3.9/copy.py", line 172, in deepcopy
    y = _reconstruct(x, memo, *rv)
  File "/home/studio-lab-user/.conda/envs/default/lib/python3.9/copy.py", line 270, in _reconstruct
    state = deepcopy(state, memo)
  File "/home/studio-lab-user/.conda/envs/default/lib/python3.9/copy.py", line 146, in deepcopy
    y = copier(x, memo)
  File "/home/studio-lab-user/.conda/envs/default/lib/python3.9/copy.py", line 230, in _deepcopy_dict
    y[deepcopy(key, memo)] = deepcopy(value, memo)
  File "/home/studio-lab-user/.conda/envs/default/lib/python3.9/copy.py", line 172, in deepcopy
    y = _reconstruct(x, memo, *rv)
  File "/home/studio-lab-user/.conda/envs/default/lib/python3.9/copy.py", line 272, in _reconstruct
    y.__setstate__(state)
  File "/home/studio-lab-user/.conda/envs/default/lib/python3.9/site-packages/insightface/model_zoo/model_zoo.py", line 33, in __setstate__
    self.__init__(model_path)
  File "/home/studio-lab-user/.conda/envs/default/lib/python3.9/site-packages/insightface/model_zoo/model_zoo.py", line 25, in __init__
    super().__init__(model_path, **kwargs)
  File "/home/studio-lab-user/.conda/envs/default/lib/python3.9/site-packages/onnxruntime/capi/onnxruntime_inference_collection.py", line 396, in __init__
    raise e
  File "/home/studio-lab-user/.conda/envs/default/lib/python3.9/site-packages/onnxruntime/capi/onnxruntime_inference_collection.py", line 383, in __init__
    self._create_inference_session(providers, provider_options, disabled_optimizers)
  File "/home/studio-lab-user/.conda/envs/default/lib/python3.9/site-packages/onnxruntime/capi/onnxruntime_inference_collection.py", line 415, in _create_inference_session
    raise ValueError(
ValueError: This ORT build has ['TensorrtExecutionProvider', 'CUDAExecutionProvider', 'CPUExecutionProvider'] enabled. Since ORT 1.9, you are required to explicitly set the providers parameter when instantiating InferenceSession. For example, onnxruntime.InferenceSession(..., providers=['TensorrtExecutionProvider', 'CUDAExecutionProvider', 'CPUExecutionProvider'], ...)
```

### Steps to reproduce the problem

1. Go to .... 
2. Press ....
3. ...

### Sysinfo

WEBUI

### Relevant console log

```Shell
Interrupted with signal 2 in <frame at 0x55944980f1a0, file '/home/studio-lab-user/.conda/envs/default/lib/python3.9/threading.py', line 316, code wait>
```

**Comment by Gourieff** [comment]
> First, confirm
I have read the [instruction](https://github.com/Gourieff/sd-webui-reactor/blob/main/README.md) carefully
I have searched the existing issues

If you confirm it, so please really do it

https://github.com/Gourieff/sd-webui-reactor#troubleshooting s.VI

https://github.com/Gourieff/sd-webui-reactor/issues?q=is%3Aissue+is%3Aclosed+This+ORT+build+has+%5B%27TensorrtExecutionProvider%27%2C+%27CUDAExecutionProvider%27%2C+%27CPUExecutionProvider%27%5D

**Comment by Gourieff** [comment]
Inside you conda env you have to delete onnxruntime-gpu and install onnxruntime
ReActor won't work with onnxruntime-gpu
Perhaps, some other extension installs onnxruntime-gpu instead of onnxruntime at every SD WebUI launch

**Comment by coudys** [comment]
> Inside you conda env you have to delete onnxruntime-gpu and install onnxruntime ReActor won't work with onnxruntime-gpu Perhaps, some other extension installs onnxruntime-gpu instead of onnxruntime at every SD WebUI launch

This was really the solution in my case. After I cleaned the venv of onnxruntime-gpu I started the web-ui only to realize the onnxruntime-gpu is present again. I had that wd14-Tagger installed which automatically installed onnxruntime-gpu every start of A1111 web-ui. Removing this extension and uninstalling onnxruntime-gpu one last time fixed for me. Which leads me to a wish, reActor would work with onnxruntime-gpu which would be faster and other extensions could coexist.
